OpenRA.Game/WorldUtils.cs

```diff
@@ -66,24 +66,16 @@ public static bool ContainsTemporaryBlocker(this World world, CPos cell, Actor i
 		public static void DoTimed<T>(this IEnumerable<T> e, Action<T> a, string text)
 		{
 			// PERF: This is a hot path and must run with minimal added overhead.
-			// Calling Stopwatch.GetTimestamp is a bit expensive, so we enumerate manually to allow us to call it only
-			// once per iteration in the normal case.
-			// See also: RunActivity
-			var longTickThresholdInStopwatchTicks = PerfTimer.LongTickThresholdInStopwatchTicks;
+			// Calling Stopwatch.GetTimestamp is a bit expensive, so we enumerate manually to allow us to call
+			// perfLogging.Log only once per iteration in the normal case.
+			var perfLogging = new SimulationPerfLogging();
```

Comment:
Likewise here, although it would need to be passed as an argument into the method.